mb-applet-battery: don't declare line_parse as static

Fix this build failure:
    mb-applet-battery.c: In function 'read_apm':
    mb-applet-battery.c:181:2: error: invalid storage class for function 'line_parse'

Reported-by: Arnout Vandecappelle <arnout@mind.be>
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN matchbox-panel-0.9.3.orig/applets/mb-applet-battery.c matchbox-panel-0.9.3/applets/mb-applet-battery.c
--- matchbox-panel-0.9.3.orig/applets/mb-applet-battery.c	2006-02-07 23:27:24.000000000 +0100
+++ matchbox-panel-0.9.3/applets/mb-applet-battery.c	2015-05-02 17:13:00.843578796 +0200
@@ -177,7 +177,7 @@
 		unsigned long	present_rate;
 	};
 
-	static int
+	int
 	line_parse(char *line, struct entry *e)
 	{
 		char *ptr;
